addons/addon-base-raas/packages/base-raas-services/lib/data-source/access-strategy/legacy/environment-resource-service.js

```diff
@@ -23,6 +23,42 @@ const { isOpenData } = require('../../../study/helpers/entities/study-methods');
 const settingKeys = {
   studyDataBucketName: 'studyDataBucketName',
   studyDataKmsKeyArn: 'studyDataKmsKeyArn',
+  studyDataKmsPolicyWorkspaceSid: 'studyDataKmsPolicyWorkspaceSid',
+};
+
+const listStatementParamsFn = (bucket, prefix) => {
+  return {
+    statementId: `List:${prefix}`,
```

Comment:
Can you check what characters are allowed to appear in the statementId?  This is because the prefix of an S3 object might also contain characters that the statement id might not allows.

Reply:
It seems like the statementId if a free-form field. I tried using all the special characters including the characters that should be avoided mentioned on S3 documentation[1] and was still able to save the policy.

https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html